packages/dnb-eufemia/src/components/upload/UploadFileListCell.tsx

```diff
@@ -168,10 +170,17 @@ const UploadFileListCell = ({
           onClick={onClick}
           bottom={0}
         />
+        {getDescription()}
       </div>
     )
   }
 
+  function getDescription() {
+    return description ? (
+      <P className="dnb-upload__text">{description}</P>
+    ) : null
```

Comment:
```suggestion
    if(!description){
      return null
    }
    return <P className="dnb-upload__text">{description}</P>
```
I tend to prefer writing the logic like this. Just because its more "isolated" and the if statement can be extended without changing "what we expect" from the function/component. But no big deal.

Reply:
This caused a lint issue. Will fix and improve this later this evening.